### PR TITLE
[FIX] l10n_fr: Account 607 french translation edited

### PR DESCRIPTION
**Pull request by yoba-odoo**
[FIX] l10n_fr: Account 607 french translation edited

Current behavior before PR:
In chart of accounts in french localization the accounts with 607 was named "Ventes de marchandises" where it should be named "Achats de marchandises"

Desired behavior after PR is merged:
The translation has been updated and now it is named correctly

opw-3536801

### CODE REVIEW

File: addons/l10n_fr/data/template/account.account-fr.csv [hunk 1]
```diff
@@ -472,8 +472,8 @@
 "pcg_6063","Maintenance and minor equipment supplies","606300","expense","account.account_tag_operating","False","Fournitures d'entretien et de petit équipement"
 "pcg_6064","Administrative supplies","606400","expense","account.account_tag_operating","False","Fournitures administratives"
 "pcg_6068","Other materials and supplies","606800","expense","account.account_tag_operating","False","Achats autres matières et fournitures"
-"pcg_6071","Goods for resale (or group) A","607100","expense","account.account_tag_operating","False","Ventes de marchandises (ou groupe) A"
-"pcg_6072","Goods for resale (or group) B","607200","expense","account.account_tag_operating","False","Ventes de marchandises (ou groupe) B"
+"pcg_6071","Purchase of goods (or group) A","607100","expense","account.account_tag_operating","False","Achat de marchandises (ou groupe) A"
```
Comment (guva-odoo) [inline addons/l10n_fr/data/template/account.account-fr.csv:475]: Should'nt it be "Achats de marchandises" ?
https://www.l-expert-comptable.com/plan-comptable/compte-607-achats-de-marchandises

Comment (yoba-odoo) [reply addons/l10n_fr/data/template/account.account-fr.csv]: Sorry, my bad. It is edited now

Comment (guva-odoo) [reply addons/l10n_fr/data/template/account.account-fr.csv]: np, thanks!

**Comment by guva-odoo** [comment]
LGTM

**Review by JulienVR — COMMENTED** [review]
I'm ok with the diff, but you should re-target this branch to saas-16.1 :)

**Comment by yoba-odoo** [comment]
> I'm ok with the diff, but you should re-target this branch to saas-16.1 :)

@JulienVR
You can check it now I have re-targeted it to saas-16.1. My fix has changed a bit because the file structure was different in this version

**Review by JulienVR — APPROVED** [review]
Thank you ! :)

**Comment by yoba-odoo** [comment]
Hello @csnauwaert ,

Could you review this please?

**Comment by csnauwaert** [comment]
@robodoo r+

**Pull request closed without merging**